nautobot_golden_config/filters.py

```diff
@@ -86,7 +82,6 @@ class GoldenConfigFilterSet(CustomFieldModelFilterSet):
     rack_group_id = TreeNodeMultipleChoiceFilter(
         queryset=RackGroup.objects.all(),
         field_name="device__rack__group",
-        lookup_expr="in",
         label="Rack group (ID)",
     )
     rack_group = django_filters.ModelMultipleChoiceFilter(
```

Comment:
should `rack_group` be a `TreeNodeMultipleChoiceFilter` as well?

Reply:
Tacking here, don't want to hold up PR for this though. Thanks for the heads up!!